actionmailbox/lib/action_mailbox/base.rb

```diff
@@ -124,4 +124,5 @@ def track_status_of_inbound_email
   end
 end
 
-ActiveSupport.run_load_hooks :action_mailbox, ActionMailbox::Base
+after_hook = defined?(Rails.application) && Rails.application ? :after_initialize : nil
```

Comment:
All those `Rails` checks should not be here. The frameworks should have no knowledge of Rails, even if it is to check for its existence. That is why they have Railties. Those checks should be moved to their Railties.

Reply:
I've moved the logic to the Railtie.